src/app.js

```diff
@@ -520,15 +520,7 @@ var run = function () {
     } else {
       url = 'https://ethereum.github.io/solc-bin/bin/' + version;
     }
-    var isFirefox = typeof InstallTrigger !== 'undefined';
```

Comment:
which firefox version are you running?
so https://ethereum.github.io/browser-solidity/#version=builtin
is working for you?